Project_Starting_Point/Server/Server.cpp

```diff
@@ -17,8 +20,69 @@ StorageTypes RxData[7];
 void UpdateData(unsigned int, float);
 float CalcAvg(unsigned int);
 
+class LatencyTimer {
+	LARGE_INTEGER frequency{ 0 };
+	LARGE_INTEGER start_t{ 0 }, end_t{ 0 }, elapsedTime{ 0 };
+	vector<pair<string, LONGLONG>> latencyList;
+	
+public:
+	LatencyTimer() : frequency{ 0 }, start_t{ 0 }, end_t{ 0 }, elapsedTime{ 0 } {};
```

Comment:
We could put the function name in the constructor here instead of at the end. This would be for `SingleLatencyEvent`